Integrations/src/main/java/io/deephaven/integrations/python/PythonMergedListenerAdapter.java

```diff
@@ -91,6 +101,19 @@ public ArrayList<TableUpdate> currentRowsAsUpdates() {
 
     @Override
     protected void process() {
-        pyCallable.call("__call__");
+        try {
+            pyListenerCallable.call("__call__");
+        } catch (Exception e) {
```

Comment:
This probably does not belong here. We should instead override `propagateErrorDownStream()` and deliver the error from there.

We *could* keep both, and have `propagateErrorDownStream()` be a no-op when `fromProcess == true`. But we *do* need that override to deliver upstream errors (which is the really important part).